crates/bevy_render/src/primitives/mod.rs

```diff
@@ -72,12 +72,46 @@ impl Sphere {
     }
 }
 
-/// A plane defined by a normal and distance value along the normal
-/// Any point p is in the plane if n.p = d
-/// For planes defining half-spaces such as for frusta, if n.p > d then p is on the positive side of the plane.
+/// A plane defined by a unit normal and distance from the origin along the normal
+/// Any point p is in the plane if n.p + d = 0
+/// For planes defining half-spaces such as for frusta, if n.p + d > 0 then p is on
+/// the positive side (inside) of the plane.
 #[derive(Clone, Copy, Debug, Default)]
 pub struct Plane {
-    pub normal_d: Vec4,
+    normal_d: Vec4,
+}
+
+impl Plane {
+    /// Constructs a `Plane` from a 4D vector whose first 3 components
+    /// are the normal and whose last component is the distance along the normal
+    /// from the origin.
+    /// This constructor ensures that the normal is normalized and the distance is
+    /// scaled accordingly so it represents the signed distance from the origin.
+    pub fn new(normal_d: Vec4) -> Self {
```

Comment:
should this one be inline too?

Reply:
I suppose it could be. I haven’t tested it. I’m just using Cart’s of using online for accessors. But I don’t think we tend to use them for constructors.

Reply:
it seems the constructor is used a few time during `from_view_projection`

Reply:
I've made the suggestion. I'm leaving it as approved as it's too minor to block on imo.